homework_sets/formHandler.js

```diff
@@ -0,0 +1,16 @@
+const formEl = document.getElementById('form');
+const inputEl = document.getElementById('input');
+
+const emoji = ['🐮', '🐯', '🐼', '🐨', '🐵', '🐷', '🐸'];
+
+export const insertRandomEmoji = () => {
+  inputEl.value = emoji[Math.floor(Math.random() * emoji.length)];
```

Comment:
It's better to separate two different functions: `generateRandomEmoji` and `setValue` (for example)